api/external/applications/applications.proto

```diff
@@ -133,53 +133,6 @@ message HealthCounts {
   int32 unknown = 5;
 }
 
-// A HabService message is the representation of one single service in habitat
-message HabService {
```

Comment:
Long live to our "**HabService**" message!
![tenor-204203012](https://user-images.githubusercontent.com/5712253/57788459-6ef57f00-7737-11e9-80d0-56fc0823f586.gif)